src/solver/conv_asm_implicit_gemm_gtc_bwd_nhwc.cpp

```diff
@@ -995,6 +998,8 @@ ConvSolution ConvAsmImplicitGemmGTCDynamicBwdXdlopsNHWC::GetSolution(
     result.construction_params[0].comp_options = opts_0.str();
     std::ostringstream msg;
 
+    msg << ", determ:" << (ctx.conv_problem.GetConv().attribute.deterministic.Get() ? "y" : "n");
```

Comment:
Do you plan to remove this?

Reply:
Actually this is to have some debug message inside solver, incase other developers want to double confirm this solver is deterministic or not

Reply:
Please move this logging into GetSolution().